spec/rails_helper.rb

```diff
@@ -46,6 +46,9 @@
 
   config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
+    # Seems cleaner to just put this here than track exactly which tests need the seed data.
+    # The Languages dependency at least is pretty fundamental
+    seed_data
```

Comment:
As written, this won't work the way you intend. `before(:suite)` only runs once, before the entire test suite, which will then be truncated for every test. I'm still really not comfortable with this though, using `seed_data` should be very rare and it's *very* useful to know which tests actually need the seed data, most shouldn't. 

In my `loans_index` branch, for instance, the call to `seed_data` for the `loans` model spec has been removed, and it also won't break anything to remove it from the `person` and `organization` model specs. It's important that we be able to keep track of where we have implicit dependencies on a specific database state, relying on it makes things brittle and seeding too much data, besides slowing down the test suite, can mask issues with the code.